### ISSUE TITLE
Locations of Oxford Nanopore fast5 data?

### ISSUE THREAD

**Issue by oneillkza** [genome-in-a-bottle/giab_data_indexes#8]
I see you now have PromethION data for several of the individuals, available in fastq format. However, I don't see a link to the raw fast5. Is this available somewhere?

**Comment by nate-d-olson** [comment]
The raw fast5s are hosted on the Human Pangenome Projects S3 bucket. Let us know if you have any questions or issues accessing the files.

HG002: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=HG002/nanopore/ 
HG003: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=HG003/nanopore/
HG004: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=HG004/nanopore/
HG005: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=HG005/nanopore/
HG006: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=HG006/nanopore/
HG007: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=HG007/nanopore/

**Comment by vahidAK** [comment]
Hi @nate-d-olson  and @jzook,

It seems that these links are not working anymore?

Do you have any idea where those files can be accessed?

Thanks,
Vahid

**Comment by nate-d-olson** [comment]
Looks like they restructured the S3 bucket here are the updated links.

HG002: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=NHGRI_UCSC_panel/HG002/nanopore/
HG003: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=NHGRI_UCSC_panel/HG003/nanopore/
HG004: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=NHGRI_UCSC_panel/HG004/nanopore/
HG005: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=NHGRI_UCSC_panel/HG005/nanopore/
HG006: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=NHGRI_UCSC_panel/HG006/nanopore/
HG007: https://s3-us-west-2.amazonaws.com/human-pangenomics/index.html?prefix=NHGRI_UCSC_panel/HG007/nanopore/

**Comment by vahidAK** [comment]
Thanks a lot

**Comment by snajder-r** [comment]
If I unpack GM24**3**85_1.fast5.tar.gz, it contains the directory GM24**1**85_1. I assume that's just a minor error in the naming of the folder, and the data is actually for GM24385?

**Comment by nate-d-olson** [comment]
@snajder-r Yes, I believe it is a typo, especially considering that I am unaware of the GM24185 cell line. I am confirming that this is a type with our collaborators at UCSC.

**Comment by nate-d-olson** [comment]
@snajder-r Our collaborators at UCSC confirmed that it is a typo. Thanks for using our data and letting us know about this typo. :)

**Comment by gambalab** [comment]
Hi,
which nanopore chemistry was used for Ulra long sequencing with Promethion? R9.4.1 or R10.4.1?

**Comment by nate-d-olson** [comment]
All the USCS Ultralong promethion data currently on the ftp site were generated using R9.4.1.
